logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/mention/MessageMention.kt

```diff
@@ -5,5 +5,8 @@ import com.wire.kalium.logic.data.user.UserId
 data class MessageMention(
     val start: Int,
     val length: Int,
-    val userId: UserId
+    val userId: UserId,
+    val mentionType: MentionType? = null
```

Comment:
Why expose this in logic module? This could be contained to the protobuf module.